sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxReportSensor.java

```diff
@@ -177,13 +177,16 @@ protected String getStringProperty(String name, String def) {
       directoryScanner.setIncludes(includes.toArray(new String[includes.size()]));
       directoryScanner.scan();
 
-      for (String found : directoryScanner.getIncludedFiles()) {
-        CxxUtils.LOG.debug("Adding report '{}'", found);
+      String [] includeFiles = directoryScanner.getIncludedFiles();
```

Comment:
instead of printing all foudn reports is best just to sumarize the number since when we parse the same reported is printed